pkg/flink/app/application.go

```diff
@@ -63,7 +63,7 @@ func StartApp(gatewayClient ccloudv2.GatewayClientInterface, tokenRefreshFunc fu
 	resultFetcher := results.NewResultFetcher(dataStore)
 
 	// Instantiate lsp
-	lspClient := lsp.NewLSPClientWS(getAuthToken, appOptions.GetLSPBaseUrl(), appOptions.GetOrganizationId(), appOptions.GetEnvironmentId())
+	lspClient := lsp.NewLSPClientWS(getAuthToken, appOptions.GetLSPBaseURL(), appOptions.GetOrganizationId(), appOptions.GetEnvironmentId())
```

Comment:
Functions of a package do not generally contain the name of the package... it's a bit redundant. Example: `http.NewClient()` instead of `http.NewHttpClient()`
```suggestion
	lspClient := lsp.NewClientWs(getAuthToken, appOptions.GetLSPBaseURL(), appOptions.GetOrganizationId(), appOptions.GetEnvironmentId())
```